components/gitpod-protocol/data/gitpod-schema.json

```diff
@@ -287,6 +287,10 @@
                                     "description": "Whether only stable, latest or both versions should be warmed up. Default is stable only."
                                 }
                             }
+                        },
+                        "vmoptions": {
+                            "type": "string",
+                            "description": "Configure JVM options, for instance '-Xmx=4096m'."
```

Comment:
We've called this `vmoptions` but from the description we're talking about JVM options. Should the name be `jvmoptions` instead?

Reply:
Hi @andrew-farries! Thanks for the review. ❤️ 

In this case, `vmoptions` is a term used by JetBrains IDE (and it's quite well-known to JB users) and I think Gitpod users (using JB Gateway) can understand its meaning as well. In fact, the [JVM Options configuration file](https://www.jetbrains.com/help/idea/tuning-the-ide.html) for intellij-based IDE is named `idea64.vmoptions` not `idea64.[j]vmoptions`. 

IMO using "JVM Options" is definitely correct for any Java application, however using "vmoptions" for JetBrains IDEs configuration is more accurate. @akosyakov also made an explanation at https://github.com/gitpod-io/gitpod/pull/10768#issuecomment-1162979895

Reply:
To sum up, `vmoptions` is used in gitpod spec within the `jetbrains` property (which is JetBrains IDE user-facing) & variable names, and `JVM Options` is used for detailed description or documentation.

Reply:
I think it is aligned with JB way.